client/src/components/ProTx/components/dashboard/DashboardDisplay.js

```diff
@@ -35,6 +36,9 @@ function DashboardDisplay() {
     ''
   );
   const [unit, setUnit] = useState('count');
+  const [map, setMap] = useState(null);
+  const [resourceLayers, setResourceLayers] = useState(null);
+
```

Comment:
@duckonomy @taoteg I had to place these states outside of the map so they could be used to figure out what needs to be downloaded.

Not the cleanest thing but didn't want to improve this more as I thought it would require more code (and wanted to avoid that as we migrate to protx-dashboard)